app/build.gradle

```diff
@@ -26,6 +26,12 @@ android {
         resolutionStrategy.force "com.android.support:animated-vector-drawable:$Versions.support"
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/worker8/Pixels/blob/43ff13c8bf415f17743a21d636736be69e6bb8df/app/build.gradle#L26"></span>All com.android.support libraries must use the exact same version specification (mixing versions can lead to runtime crashes). Found versions 28.0.0, 27.1.0. Examples include <code>com.android.support:animated-vector-drawable:28.0.0</code> and <code>com.android.support:exifinterface:27.1.0</code></td>
    </tr>
  </tbody>
</table>